pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -815,28 +815,77 @@ cdef class _Timedelta(timedelta):
 
     cpdef timedelta to_pytimedelta(_Timedelta self):
         """
-        return an actual datetime.timedelta object
-        note: we lose nanosecond resolution if any
+        Converts a Timedelta object into a python datetime.timedelta object.
+
+        Timedelta objects are typecast to numpy datetime64[ns] dtype,
+        use to_pytimedelta() to convert back to object dtype.
+
+        Returns
+        -------
+        datetime.timedelta or numpy.array of datetime.timedelta
+
+        See Also
+        --------
+        to_timedelta : Convert argument to Timedelta type.
+
+        Notes
+        -----
+        Any nanosecond resolution will be lost.
+
+        Examples
+        --------
```

Comment:
not sure i would add all of these examples, 1 example at most and it should actually use this function

Reply:
@jreback 
I've made these changes: https://github.com/pandas-dev/pandas/pull/25118/commits/c09d83a1b1368c0101f9992108811cc4ccfd466c
If there's anything else, please let me know.

Reply:
Do these doctests pass? It looks like the whitespace formatting is off.

You may need to add a `doctest: +NORMALIZE_WHITESPACE`

Reply:
I don't think any of these examples are needed, they are not showing *this* internal rountine. I would simply remove them.

Reply:
@dcreekp as I said above I would remove all of these examples